tls/s2n_resume.c

```diff
@@ -465,7 +471,7 @@ S2N_RESULT s2n_connection_get_session_state_size(struct s2n_connection *conn, si
     RESULT_ENSURE_REF(conn);
     RESULT_ENSURE_REF(state_size);
 
-    if (conn->actual_protocol_version < S2N_TLS13) {
+    if (conn->actual_protocol_version < S2N_TLS13 || !IS_NEGOTIATED(conn)) {
```

Comment:
Just to make sure-- do the tests fail without this new check?

Reply:
Removed this check.